bundles/pubsub/pubsub_admin_websocket/src/pubsub_websocket_topic_receiver.c

```diff
@@ -558,34 +586,88 @@ static void* psa_websocket_recvThread(void * data) {
     return NULL;
 }
 
+static void psa_websocketTopicReceiver_ready(struct mg_connection *connection, void *handle) {
+    if (handle != NULL) {
```

Comment:
You can reduce nesting by changing this to `if(handle == null) { return; }`

Reply:
Changing that will decrease the coverage